sdk/scala-sdk/src/main/scala/kalix/scalasdk/impl/action/ActionEffectImpl.scala

```diff
@@ -112,6 +111,16 @@ private[scalasdk] object ActionEffectImpl {
     }
   }
 
+  final case class IgnoreEffect[T](internalSideEffects: Seq[SideEffect]) extends PrimaryEffect[T] {
+    def isEmpty: Boolean = true
+    protected def withSideEffects(sideEffect: Seq[SideEffect]): IgnoreEffect[T] =
+      copy(internalSideEffects = sideEffect)
```

Comment:
Same fail early here to get the call site where you call it in the stack trace rather than somewhere else in internal code where we toJavaSdk it?

Reply:
right